src/backend/routers/chat.py

```diff
@@ -165,7 +162,6 @@ async def chat(
         CustomChat().chat(
             chat_request,
             stream=False,
-            file_paths=file_paths,
```

Comment:
I'll follow up with a PR to remove file path everywhere. Tracking here https://linear.app/cohereai/issue/TLK-967/remove-file-paths-from-all-be-code

Reply:
#652 now does this 🥇